crates/tx-graph/src/peg_out_graph.rs

```diff
@@ -546,6 +559,8 @@ impl PegOutGraphConnectors {
     }
 }
 
+type PegOutGraphSighashes = PogMusigF<Message>;
```

Comment:
Are we exporting this type? If yes, might be good to add docstrings

Reply:
Honestly I'm just gonna ditch it.